processing/bloom_handler.go

```diff
@@ -165,7 +160,8 @@ func MakeBloomHandlerFromFile(bloomFilename string, compressed bool,
 			log.Warnf("file is empty, using empty default one")
 			myBloom := bloom.Initialize(100, 0.00000001)
 			iocBloom = &myBloom
-		} else if strings.Contains(err.Error(), "value of k (number of hash functions) is too high") {
+		} else if strings.Contains(err.Error(), "value of k (number of hash "+
```

Comment:
I was looking for what was changed here, and looking at the rest: this is just a "line-too-long"? It's 2020 <80 is absolutely over, since a decade now. Time times we did live patching in server rooms, in the basements, on CRTs is over.

Reply:
This is basically it. Usually I try to adhere to 80 chars per line and was just clearing it up here as I scrolled over the code to make the other adjustments. Since these static strings are likely concatenated at compile time, I don't see an actual impact here, but I do acknowledge that this adds extra noise in PRs. Do you expect these changes to be removed to merge this?

Reply:
Reverted these changes for cleanliness.